src/Database/Log/QueryLogger.php

```diff
@@ -47,7 +65,8 @@ public function log(LoggedQuery $query)
      */
     protected function _log($query)
     {
-        Log::write('debug', $query, ['queriesLog']);
+        $message = "connection_id={$this->_connectionId} " . $query->__toString();
```

Comment:
If the connection id is null could this be skipped?

Reply:
Of course.